src/commands/init.ts

```diff
@@ -141,10 +95,77 @@ export class InitCommand implements Command {
           };
           inquirer.prompt([prompt]).then((answers) => {
             let generatorName = answers.generatorName;
-            runGenerator(generatorName, getDisplayName(generatorName));
+            runGenerator(generatorName, InitCommand.getDisplayName(generatorName));
           });
         }
       });
     });
   }
+
+  private static registerDefaultGenerators(env: YeomanEnvironment): void {
```

Comment:
Yeah, I originally did that so that I could test them independently of the main run method, before I figured out how to mock the imported modules. However, I'm thinking I may go back and restructure the tests so that just go through run, now that I've figured out all of that mocking and such.